build/Packaging.cake

```diff
@@ -2,17 +2,27 @@
 
 var version = Argument("version", "0.4.0");
 
+Task("Build-ObjC")
+    .Does(() =>
+    {
+        var exitCode = StartProcess("make", new ProcessSettings {
```

Comment:
That raises an interesting question. If I do that, and you build a nuget on Windows, it will differ than macOS.

Should we make building all of objc safe on Windows? Post a warning?

Reply:
I think if you build on windows you might only care about the android bits, I see this for rapid iteration when using a local NuGet package and a project using E4K, so initially I guess we can get away just building the full NuGet in macOS for our publishing needs